Update README.md (#204)
ava-labs/avalanche-rs: README.md

> **This repository is no longer maintained. If you are interested in maintaining it, please reach out in the discussion tab.**

Reviewer: ```suggestion
**This repository is not currently maintained. If you are interested in maintaining it, please reach out in the discussion tab.**
```
